### PR TITLE
Add various thoughts

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: draft-fossati-cose-profiles.md [hunk 1]
```diff
@@ -80,9 +82,29 @@ A COSE profile:
     * using an OID {{-oid}}, URI {{-uri}} or CRI {{-cri}}
     * using a UUID {{-uuid}}
   * The chosen value SHOULD be appropriate for the intended usage scope (e.g., a small value when used in constrained node environments)
-* MAY define its own COSE tag
+* MAY define its own CBOR tag
```
Comment (henkbirkholz) [inline draft-fossati-cose-profiles.md]: we should probably get even more specific here (in later iterations)

File: draft-fossati-cose-profiles.md [hunk 2]
```diff
@@ -91,6 +113,12 @@ TODO Security
 
 This document has no IANA actions.
```
Comment (henkbirkholz) [inline draft-fossati-cose-profiles.md]:
```suggestion
```

File: draft-fossati-cose-profiles.md [hunk 3]
```diff
@@ -91,6 +113,12 @@ TODO Security
...
 
+Actually, it defines a header parameter "COSE_profile" that needs to
```
Comment (henkbirkholz) [inline draft-fossati-cose-profiles.md]:
```suggestion
This memo defines a header parameter "cose-profile" that needs to
```

File: draft-fossati-cose-profiles.md [hunk 4]
```diff
@@ -91,6 +113,12 @@ TODO Security
...
+be registered in the "COSE Header Parameters" registry.
+
+(Note that parameter names usually don't have programming language
+lexicals, i.e., no underscores etc.)
+
```
Comment (henkbirkholz) [inline draft-fossati-cose-profiles.md]: worth replacing right away

**Review by henkbirkholz — CHANGES_REQUESTED** [review]

**Comment by SteveLasker** [comment]
Thankss @cabo for the PR

File: draft-fossati-cose-profiles.md (hunk 3 above)
Comment (henkbirkholz) [reply draft-fossati-cose-profiles.md]: Anticipating #2, I am changing this here already, too.

**Review by thomas-fossati — APPROVED** [review]
thanks!

File: draft-fossati-cose-profiles.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-fossati-cose-profiles.md]: would you mind raising an issue to track future actions?

File: draft-fossati-cose-profiles.md (hunk 4 above)
Comment (thomas-fossati) [reply draft-fossati-cose-profiles.md]: I see you've done that in #2

**Review by letmaik — APPROVED** [review]

**Pull request merged**